ansible/roles/stack-sunbird/templates/uci_api.env

```diff
@@ -0,0 +1,19 @@
+ENV=prod
+DATABASE_DEBUG=true
+HASURA_GRAPHQL_ADMIN_SECRET=HaTUNsNSaU
```

Comment:
No default password please

Reply:
Changed to HASURA_GRAPHQL_ADMIN_SECRET={{hasura_graphql_admin_secret}}, as discussed in #2702